### PR TITLE
Removes all the SDK images and the qt5 sdk class

### PR DESCRIPTION
**Pull request by tobsan**
This way, the class is only loaded if meta-qt5 is also present. Otherwise, the classes that this class inherits won't be available.

### CODE REVIEW

**Comment by socec** [comment]
Shouldn't we also remove the inherit statement from core-image-bistro-sdk-1.1.bb since it could happen that you do not include the qt layer?

**Comment by Gagi2k** [comment]
I would prefer to remove this file completely as we reuse whatever provides meta-boot2qt nowadays

**Comment by tobsan** [comment]
Oh yes, that should definitely be done, I'll update the MR. In my setup I removed those image recipes for completely different reasons, so I didn't see that problem.

**Comment by tobsan** [comment]
Commenting on @Gagi2k, I would agree if meta-bistro depended on meta-boot2qt, but it doesn't. 

Actually, that makes the whole image thing strange since we would include a class which would be  provided upstream. But as @rpannek pointed out in #32 we should probably remove that image recipe anyway.

**Comment by socec** [comment]
I would like to provide only a single demo/base image and remove all others. AFAIK, this layer is meant to be an extension of meta-ivi and the actual variety of images will be provided in the top layer of the project that will want to use meta-ivi and meta-bistro.

File: layers/qt5-layer/classes/populate_sdk_qt5_qtcreator.bbclass [hunk 1]
```diff
@@ -1,125 +0,0 @@
...
-
-        if [[ $success -ne 0 ]]; then
-            echo "Removed Kit successfully"
-        fi
-fi
-EOF
-
-	chmod +x ${QTCREATORSCRIPTPATH}
-}
-
```
Comment (rpannek) [inline layers/qt5-layer/classes/populate_sdk_qt5_qtcreator.bbclass]: By removing this class we are breaking code of at least one of our customers, next time we should perhaps give them a heads up.

Comment (Gagi2k) [inline layers/qt5-layer/classes/populate_sdk_qt5_qtcreator.bbclass]: but they are hopefully not on the master branch and are also on a fixed sha1 ?

Comment (rpannek) [inline layers/qt5-layer/classes/populate_sdk_qt5_qtcreator.bbclass]: I don't remember, I'll have to take a look tomorrow.

Comment (rpannek) [inline layers/qt5-layer/classes/populate_sdk_qt5_qtcreator.bbclass]: They follow the jethro branch.

Comment (Gagi2k) [inline layers/qt5-layer/classes/populate_sdk_qt5_qtcreator.bbclass]: then i guess it's ok as the change is in the master branch...

**Pull request merged**